src/services/Where.ts

```diff
@@ -129,6 +129,8 @@ export class Where {
           queries.push(`(${Where.parseQueryValue(subkey, value[subkey])})`);
         }
         return `(${queries.join(' OR ')})`;
+      } else if (key.startsWith('group_')) {
```

Comment:
What do we use the part after the group_ for? Is there a benefit to having something after the prefix?

Reply:
It's so we can create these grouped sections like in the test example I added:
        group_1: {
          startDate: { or: {max: '2022-01-01', isNull: true}},
        },
        group_2: {
          endDate: { or: { min: '2026-12-31', isNull: true }},
        }

This will get me a result of (startDate: IS NULL OR startDate < \'2022-01-01\') AND (endDate: IS NULL OR endDate >= \'2026-12-31\')

Reply:
Oh, I get it. We're dealing with an object here, so we have to account for object properties not allowing the same name. That's less desirable than a true nested query structure where you simply have: 'group'. But I get why you have to do that.